README.md

```diff
@@ -162,7 +162,7 @@ For those not interested in operating and maintaining a fleet of Ship instances,
 
 # Community
 
-For questions about using Ship, there's a [Replicated Community](https://help.replicated.com/community) forum.
+For questions about using Ship, there's a [Replicated Community](https://help.replicated.com/community) forum, and a [#ship channel in Kubernetes Slack](https://kubernetes.slack.com/messages/CFFKXV0JZ/).
```

Comment:
Should we use a channel link as described here: https://stackoverflow.com/a/37187792/5657035 instead?

Reply:
Good call!  Updated